okapi/core/CronJob/CacheCleanupCronJob.php

```diff
@@ -49,27 +50,27 @@ public function execute()
             where
                 c.`key` = cr.cache_key
                 and c.score is not null
-        ");
-        Db::execute("truncate okapi_cache_reads");
+        ');
+        Db::execute('truncate okapi_cache_reads');
 
-        # Delete elements with the lowest score. Entries which have been set
-        # but never read will be removed after 2 days (0.08^2 < 0.01 < 0.08^1).
+        // Delete elements with the lowest score. Entries which have been set
+        // but never read will be removed after 2 days (0.08^2 < 0.01 < 0.08^1).
 
         Db::execute("
             delete from okapi_cache
             where
                 score is not null
                 and score < '".Db::escape_string($limit)."'
         ");
-        Db::query("optimize table okapi_cache");
+        Db::query('optimize table okapi_cache');
```

Comment:
No. SQLs should be consistently quoted, not some in double quotes and some in single quotes.